package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gemini-weather",
+  "version": "1.0.0",
+  "description": "gemini weather app",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "concurrently \"nodemon server.js\" \"cd ./client && npm start\"",
```

Comment:
create a dev script and move the start to dev.

Reply:
done :)